src/Network/JSONApi/Document.hs

```diff
@@ -17,6 +17,7 @@ module Network.JSONApi.Document
   , mkIncludedResource
   ) where
 
+import Control.DeepSeq (NFData)
```

Comment:
@gwils this PR does not make it quite clear why NFData is required?

Reply:
NFData is for forcing evaluation. Haskell is lazy by default, but sometimes we want to force evaluation.
For example, I'm adding NFData here to use in some benchmarks I'm writing.